DataFormats/L1TGlobal/interface/GlobalAlgBlk.h

```diff
@@ -109,9 +101,12 @@ class GlobalAlgBlk
     //Prescale Column
     int m_preScColumn;
 
+    //Menu UUID
+    int m_menuUUID;
+    int m_firmwareUUID;
    
     std::vector<bool> m_algoDecisionInitial;
-    std::vector<bool> m_algoDecisionPreScaled;
+    std::vector<bool> m_algoDecisionInterm;
```

Comment:
out of curiosity, what is the purpose of this change ?
does it mean that the decision after prescales but before masking is no longer available ?